packages/bp/src/admin/workspace/collaborators/collaborators-router.ts

```diff
@@ -41,15 +41,36 @@ class CollaboratorsRouter extends CustomAdminRouter {
       this.needPermissions('read', this.resource),
       this.asyncMiddleware(async (req, res) => {
         const filterRoles = req.query.roles && req.query.roles.split(',')
+        const filterAuthStrategies: string[] | undefined =
+          req.query.authStrategies && req.query.authStrategies.split(',')
```

Comment:
This should come from the current workspace and not a query param.
pseudo should look like the following:

```ts
const workspaceId = req.workspace
const authStrategies = (await getWorkspace(workspaceId)).authStrategies || []
const allUsers = await getAllUsers()
const currentWorkspaceUsers = await getWorkspaceUsers()

const availableUsers = allUsers
.filter(u => authStrategies.length ?  authStrategies.includes(u.authStrategy) : u)
.filter(u => filterRoles.length ? filterRoles.includes(u.role): u)

return availableUsers
```